common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -251,6 +335,11 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
     Utils.byteStringAsMb(get(key, defaultValue))
   }
 
+  /** Get a size parameter as mb for a transport config */
+  def getTransportConfSizeAsMb(module: String, configEntry: ConfigEntry[Long]): Long = {
```

Comment:
This method is not used too, I think adding a new method in time will be better.

Reply:
Done